task-service/openapi_user.yaml

```diff
@@ -0,0 +1,134 @@
+openapi: 3.0.3
+info:
+  title: User API
+  description: API for managing users
+  version: 0.0.1
+servers:
+  - url: '{host}/api'
+    variables:
+      host:
+        default: http://localhost:8080
+paths:
+  /users/v1:
+    get:
+      summary: Get all users
+      description: Retrieve a list of all users
+      responses:
+        '200':
+          description: A list of all users
+          parameters:
+            - in: query
+              required: false
+              name: directory
+              schema:
+                $ref: '#/components/schemas/directory'
+              description: Filter users by directory
+            - in: query
+              required: false
+              name: task
+              schema:
+                $ref: '#/components/schemas/task'
+              description: Filter users by task
```

Comment:
what does this mean? you want to get the users how have access to the task? or get the user who the task is assigned to?